bundles/org.openhab.binding.feed/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -41,6 +41,11 @@
 				<default>20</default>
 			</parameter>
 
+			<parameter name="number-of-entries" type="integer">
```

Comment:
```suggestion
			<parameter name="number-of-entries" type="integer" min="0">
```

Do you need to set a maximum value, too?

Reply:
I don't think so, as I don't want to limit the user.